packages/karma/test/karma/karma.conf.js.golden

```diff
@@ -0,0 +1,454 @@
+// Karma configuration
```

Comment:
looks brittle that this golden is so long, probably more than we meant to assert on in this case.

I'd like to make a convenience rule for `golden_content_test` or sth so we can just say "make sure this line (± whitespace) appears in the file"

this is okay for now if the content is stable

Reply:
It is stable. Should I file a bug about (perhaps) amending generated_file_test to include substring match capability?

Reply:
I have added that capability. PTAL.